android/capacitor/src/main/java/com/getcapacitor/Bridge.java

```diff
@@ -92,7 +92,7 @@ public class Bridge {
     public static final String CAPACITOR_FILE_START = "/_capacitor_file_";
     public static final String CAPACITOR_CONTENT_START = "/_capacitor_content_";
     public static final String CAPACITOR_HTTP_INTERCEPTOR_START = "/_capacitor_http_interceptor_";
-    public static final String CAPACITOR_HTTPS_INTERCEPTOR_START = "/_capacitor_https_interceptor_";
```

Comment:
Since it's public, removing it would be a breaking change, so better mark it as deprecated and we will remove it in next major.

Reply:
Done via: ac97cd4